src/scripts/index.ts

```diff
@@ -32,17 +55,17 @@ const getMajorCities: NodeListOf<HTMLElement> =
 const majorCities: HTMLElement[] = Array.from(getMajorCities);
 
 for (const city of majorCities) {
-  city.addEventListener("click", function (e) {
-    getCityCoords(city.innerText);
+  city.addEventListener("click", function () {
+    const cityData = getCityCoords(city.innerText);
```

Comment:
Similar to my comment about line 43, would not be simpler to just have ` currentCity = getCityCoords(city.innerText); ` rather than creating const valuable that will just have its value assigned to currentCity just a few lines down?

Reply:
Yes, so I plan to do more verbose error handling if I get a chance... so currentCity is always guaranteed to have a value. 
So the separation is so that when I do the error handling, if cityData is null, I'll have a pop up to alert the user of the error but not crash the site.

Reply:
Oh I see, you need currentCity to always have value as some sort of a backup. Understood 🤝